src/mongoose_hooks.erl

```diff
@@ -17,3 +67,98 @@ privacy_check_packet(LServer, Acc, User, PrivacyList, FromToNameType, Dir) ->
                              PrivacyList,
                              FromToNameType,
                              Dir]).
+
+-spec privacy_get_user_list(Server, UserList, User) -> Result when
+    Server :: jid:server(),
+    UserList :: any(),
+    User :: jid:user(),
+    Result :: mongoose_privacy:userlist().
+privacy_get_user_list(Server, UserList, User) ->
+    ejabberd_hooks:run_fold(privacy_get_user_list, Server,
+                                          UserList, [User, Server]).
+
+-spec roster_in_subscription(LServer, Acc, User, Server, From, Type, Reason) -> Result when
+    LServer :: jid:lserver(),
+    Acc :: mongoose_acc:t(),
+    User :: jid:user(),
+    Server :: jid:server(),
+    From :: jid:jid(),
+    Type :: any(),
```

Comment:
Yes, it seems it is. I was mistaken because handler `check_in_subscription` in `ejabberd_sm` expects `any()` there. This is a general question: If we know the specific type but the values are discarded, should we include this knowledge in specs? I. e. should I change spec of `check_in_subscription` to reflect the real type, even though it is not strictly connected to the logic of this function itself?